common/test/acceptance/pages/lms/discussion.py

```diff
@@ -567,6 +567,12 @@ def click_element(self, selector):
         )
         self._find_within(selector).click()
 
+    def new_post_button_visible(self):
```

Comment:
This property is returning the button actually, and is used for other purposes like clicking it. If you think it will be a good approach to check button's visibility using this property please let me know.
Thanks.